game/Bird.java

```diff
@@ -124,11 +124,11 @@ public void update() {
         if (Resources.IN_TUBE) {
             int leftBird = Resources.BIRD_X_POSITION;
             int rightBird = Resources.BIRD_X_POSITION + Resources.BIRD_WIDTH;
-            int leftTube = (int) Resources.CURRENT_TUBE.getX();
-            int rightTube = (int) Resources.CURRENT_TUBE.getX() + Resources.TUBE_WIDTH;
+            int leftTube = (int) Resources.CURRENT_TUBE.getX()+50;
```

Comment:
Have you run the code on your machine? This bug is interesting and its maybe just on my machine. Birds really shouldnt collide with thin air. :)